docs/adr/0017-python-anlaytics-processing.md

```diff
@@ -0,0 +1,205 @@
+# Python Analytics Processing
+
+## Status: Proposed
+
+## Context
+
+Our team has developed a natural language processing model for the identifying duplicate goals using python. We need to productionize this model and deploy it to our cloud infrastructure on cloud.gov, which has specific technical limitations. We have two design options to consider - containerizing the Python code and deploying it using cloud.govs underlining technology through Cloud Foundry's Docker support, or using Node child processes to wrap execution Python code on one of the workers.
```

Comment:
Not sure if you want to break this up a bit: 

From:
We have two design options to consider - containerizing the Python code and deploying it using cloud.govs underlining technology through Cloud Foundry's Docker support, or using Node child processes to wrap execution Python code on one of the workers.

To: 
We have two design options to consider - The first is containerizing the Python code and deploying it using cloud.gov's underlining technology in Cloud Foundry. The second option would be to spawn a node child processes to execute the python code.

or something with this breakdown